src/schema/validator.js

```diff
@@ -55,6 +56,27 @@ validator.addFormat(
   imageDataOrStrictRelativeUrl
 );
 
+validator.addKeyword('deprecated', {
+  validate: function validateDeprecated(message, propValue, schema, dataPath) {
+    if (!STATIC_THEME_LWT_ALIASES.includes(dataPath)) {
```

Comment:
At line 36 of this file, we are defining a "deprecated" format, but `deprecated` is actually a keyword and so it looks that we never actually took into account any of the "deprecated" keywords used in the schema files.

To avoid regressions related to new unexpected deprecated warnings/errors messages, this PR is currently ignoring any deprecation that is not coming from the static theme LWT aliases's data paths.

I'll file a follow up so that we can discuss and evaluate in more detail how to allow the deprecated keyword to work on the rest of the API schemas.

Reply:
I filed #2462 to follow up on this.